helm/private/resolved_toolchain.bzl

```diff
@@ -0,0 +1,26 @@
+"""This module implements an alias rule to the resolved helm toolchain.
+"""
+
+DOC = """\
+Exposes a concrete toolchain which is the result of Bazel resolving the
+toolchain for the execution or target platform.
+Workaround for https://github.com/bazelbuild/bazel/issues/14009
+"""
+
+# Forward all the providers
+def _resolved_toolchain_impl(ctx):
+    toolchain_info = ctx.toolchains["@rules_helm//helm:toolchain_type"]
+    return [
+        toolchain_info,
+        toolchain_info.default,
+        toolchain_info.template_variables,
+    ]
+
+# Copied from java_toolchain_alias
+# https://cs.opensource.google/bazel/bazel/+/master:tools/jdk/java_toolchain_alias.bzl
+resolved_toolchain = rule(
```

Comment:
nit: I would call this rule `current_helm_toolchain` to match the `rules_rust` pattern.

Reply:
Updated the name.